examples/alignment/tracking/tracking_kf_alignment.lcsim

```diff
@@ -0,0 +1,188 @@
+          <dataRelationCollection>""</dataRelationCollection>
+
+          <chi2Cut>9999</chi2Cut>
+        </driver>
+
+        <driver name="CleanupDriver" type="org.lcsim.recon.tracking.digitization.sisim.config.ReadoutCleanupDriver"/>
+
+        <driver name="MultEvtVtx" type="org.hps.recon.vertexing.MultipleEventsVertexingDriver">
+          <ntrks>100</ntrks>
+        </driver>
+
+        <driver name="KalmanPatRecDriver" type="org.hps.recon.tracking.kalman.KalmanPatRecDriver">
+          <!--<doDebugPlots>false</doDebugPlots>-->
+          <!-- <siHitsLimit>50</siHitsLimit> -->
+          <seedCompThr>0.05</seedCompThr>
+          <addResiduals>true</addResiduals>
+          <verbose>false</verbose>
+        </driver>
+        
+        <!-- do front-back kink plots -->
+        <driver name="KalmanKinkFitDriver" type="org.hps.recon.tracking.kalman.KalmanKinkFitDriver">
+        </driver>
+
+        <!-- Form trajectories for MPII using the GBL algorithm, starting from Kalman Tracks -->
+        <driver name="KalmanToGBLDriver" type="org.hps.recon.tracking.gbl.KalmanToGBLDriver">
+          <!--<debug>true</debug>-->
+        </driver>
+        
+        <!-- Form trajectories for MPII using the GBL algorithm -->
+        <driver name="SimpleGBLTrajAliDriverKF" 
+                type="org.hps.recon.tracking.gbl.SimpleGBLTrajAliDriver" >
+
+          <!-- apply the track quality cuts -->
+          <enableAlignmentCuts>true</enableAlignmentCuts> 
+          <!--
+          <doCOMAlignment>true</doCOMAlignment>
+          <minMom>0.5</minMom> 
+          <maxMom>5.0</maxMom> 
+          -->
+          <nHitsCut>6</nHitsCut> <!-- minimum number of hits to be included -->
+
+          <debugAlignmentDs>false</debugAlignmentDs>
+
+          <correctTrack>true</correctTrack> <!-- refit with GBL before doing Mille -->
+          <includeNoHitScatters>false</includeNoHitScatters>
+          <gblRefitIterations>0</gblRefitIterations>
+          <storeTrackStates>true</storeTrackStates>
+          <compositeAlign>true</compositeAlign>
+
+          <!-- <momC>4.55</momC> momentum constraint [GeV] -->
+          <constrainedFit>false</constrainedFit> <!-- apply momentum constraint -->
+
+          <constrainedBSFit>false</constrainedBSFit> <!-- apply beam spot constraint -->
+          <!-- <bsZ>-7.7</bsZ> beam spot z-coordinate, used to calculate beam spot -->
+
+          <trackSide>-1</trackSide> <!--hole-->
+
+          <writeMilleBinary>true</writeMilleBinary>
+          <milleBinaryFileName>${outputFile}.bin</milleBinaryFileName>
+          <writeMilleChi2Cut>5</writeMilleChi2Cut>  <!-- max Chi2/Ndf to be included -->
+
+          <enableStandardCuts>false</enableStandardCuts>
+          <maxTrackChisq4hits>60.</maxTrackChisq4hits>
+          <maxTrackChisq5hits>60.</maxTrackChisq5hits>
+          <maxTrackChisq6hits>60.</maxTrackChisq6hits>
+
+          <inputCollectionName>KalmanFullTracks</inputCollectionName>
+        </driver> 
```

Comment:
Just highlighting the area in the steering file that I should update to be CLI parameters that can then be set by the track_align job component in hps-mc.

Reply:
Also, after running this steering file a few times now, it doesn't appear to be making all the plots the steering files I got from PF make. The two most notable ones are EoP and the kinks in the tracks.